ethereum/contracts/SwapCreator.sol

```diff
@@ -1,61 +1,88 @@
 // SPDX-License-Identifier: LGPLv3
 pragma solidity ^0.8.19;
 
-import {IERC20} from "./IERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {Secp256k1} from "./Secp256k1.sol";
 
+// SwapCreator facilitates swapping between Alice, a party that has ETH or an
+// ERC-20 token that she wants to exchange cross-chain for Monero (XMR), and
+// Bob, a party that has XMR and wishes to exchange it for Alice's ETH or token.
```

Comment:
remove xmr mentions - the contract itself is generic, the other chain can be anything

Reply:
Done. I removed the references to ETH as well, as the contract can most likely be deployed as-is on BNB Smart Chain, and the Go changes required wouldn't be that extensive either. Infrastructure changes would be minimal once our bootnodes work across multiple chains.